src/mouse-keyboard-vr-display.js

```diff
@@ -161,6 +164,28 @@ MouseKeyboardVRDisplay.prototype.onMouseUp_ = function(e) {
   this.isDragging_ = false;
 };
 
+
+MouseKeyboardVRDisplay.prototype.requestPointerLock_ = function(element) {
+  // from pointerlock controls in three.js
+  // - https://github.com/mrdoob/three.js/blob/master/examples/misc_controls_pointerlock.html
+  element.requestPointerLock = element.requestPointerLock || element.mozRequestPointerLock || element.webkitRequestPointerLock;
+  if ( /Firefox/i.test( navigator.userAgent ) ) {
+    var fullscreenchange = function ( event ) {
+      if ( document.fullscreenElement === element || document.mozFullScreenElement === element ) {
+        document.removeEventListener( 'fullscreenchange', fullscreenchange );
+        document.removeEventListener( 'mozfullscreenchange', fullscreenchange );
+        element.requestPointerLock();
+      }
+    };
+    document.addEventListener( 'fullscreenchange', fullscreenchange, false );
+    document.addEventListener( 'mozfullscreenchange', fullscreenchange, false );
+    element.requestFullscreen = element.requestFullscreen || element.mozRequestFullscreen || element.mozRequestFullScreen || element.webkitRequestFullscreen;
```

Comment:
again, can remove `element.mozRequestFullscreen` because we originally got the casing wrong (that is, this never existed)